CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # jsonld-request ChangeLog
 
+## 1.0.0 - 2022-xx-xx
+
+## Changed
+- Update dependencies.
+- Modernize.
+- **BREAKING**: Convert to module.
+- **BREAKING**: Switch from callbacks to `async`/`await`.
+- **BREAKING**: Switch from deprecated `request` to `got`. Any/all options
```

Comment:
Can we use `@digitalbazaar/http-client` instead? If not now -- can we switch later without another breaking change?

Reply:
Changed.